### PR TITLE
20 video upload

### PR DESCRIPTION
**Pull request by yanivyehu**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /.nyc_output/
 /coverage/
 /node_modules/
+*Key.json
```
Comment (israeldeweb) [inline .gitignore:5]: Consider our env file for this, if the value is not too ugly and the process not too much of a hassle.

File: src/routes.js [hunk 2]
```diff
@@ -1,7 +1,14 @@
 import * as fs from 'fs';
 
+import { Storage } from '@google-cloud/storage';
+import multer from 'fastify-multer';
+
 import * as db from './db.js';
 
+const storage = new Storage({projectId: process.env.GCP_PROJECT_ID});
```
Comment (israeldeweb) [inline src/routes.js]: This, and the relevant imports, should at some point become another module (video or storage, I'm not yet certain).

**Review by israeldeweb — COMMENTED** [review]
Releasing early.

File: src/routes.js [hunk 3]
```diff
@@ -55,4 +63,37 @@ export default async(fastify, _) => {
         }
         return response.code(404).type('text/plain').send('file not found');
     });
+
+    // Upload video page
+    fastify.get('/uploadpage', async(request, response) => {
```
Comment (israeldeweb) [inline src/routes.js]: I'm thinking we might want the html file to sit in the `site` dir and be served from the a `*.html` endpoint, like our js file(s). What do you think?

Comment (yanivyehu) [reply src/routes.js]: i'm not sure we will really have endpoint of 'uploadpage'. this was just for the poc. it is temporary anyway.

File: .gitignore (hunk 1 above)
Comment (yanivyehu) [reply .gitignore:5]: let's keep it as is for now.

File: src/routes.js (hunk 3 above)
Comment (israeldeweb) [reply src/routes.js]: We will serve the upload UI from somewhere. But we might do all the UIs from a single html 🤔

Comment (yanivyehu) [reply src/routes.js]: got ya

File: package.json [hunk 4]
```diff
@@ -8,10 +8,11 @@
   },
   "scripts": {
     "refresh-db": "node -e '(async() => {await (await import(\"./db.js\")).refreshDatabase(); process.exit(0);})()'",
-    "start": "npx fastify start ./src/routes.js",
+    "start": "npm run lint; npx fastify start ./src/routes.js",
```
Comment (israeldeweb) [inline package.json]: Shouldn't this `;` be `&&`?

Comment (yanivyehu) [reply package.json]: yes! if we want to terminate on error

**Pull request merged**